geometry_msgs/Vector3 pos
geometry_msgs/Vector3 angles

float64 lat
float64 lon
float64 heading
int8 globalPos

float32 charge

uint32 missionHash
float32 height
float32 tolerance
float32 density
uint32 cloudSize

int8 armed
int8 systemState